src/guides/share/groups/customize.rst

```diff
@@ -0,0 +1,63 @@
+.. _group-customization:
+
+===========================
+Customize your group's page
+===========================
+
+.. hint::
```

Comment:
We should mention that the user has to have owner permissions, as per

> Owners ... same permissions as editors plus the ability to manage the group's description and logo

Or relax this requirement -- I think editors should be able to edit the group's customisations.

Reply:
Ah, good point. Added a mention in 3702799.

I don't agree about editors (at least currently) but am open to discussing roles more, esp. with the broader team. The current permissions gap between editors and owners is intentionally big so that there's room to add new roles between/adjacent to them in the future without breaking people's existing memberships. e.g. imagine a `managers` role that gets `editors` + customizing + membership management but not deletion of the group, or something similar. or a `curators` role that gets customization-only. Tightly-scoped roles now mean we can more easily change later. (and roles don't even have to be mutually exclusive, possibly shouldn't be, which is served better by tightly-scoped roles.)